src/parsers.nim

```diff
@@ -184,7 +184,8 @@ let
 
     NewLine = preceded(+(sp0 > p"\n"), sp0) @ (it => it.len)
 
-    Comment = preceded(s"#", p".*")
+    Comment = preceded(s"#", p".*") @ (it => newComment(it))
+    DocStr = preceded(s"#", p".*") @ (it => newComment(it, true))
```

Comment:
Do we need to distinguish between comments and documents?
```suggestion
Comment = alt(
  preceded(s"#", p".*") @ (it => newComment(it)),
  preceded(s"#", p".*") @ (it => newComment(it, true))
```

Reply:
il.comment has property "isDoc", so i thought we need to do.